src/components/select-custom/select-custom.component.html

```diff
@@ -0,0 +1,51 @@
+<button type="button"
+        class="select-button"
+        [uxMenuTriggerFor]="menu"
+        (click)="onToggle()">
+
+    <ng-container [ngTemplateOutlet]="buttonContentRef"></ng-container>
+
+    <div class="ux-select-icons">
+        <i class="ux-select-icon ux-icon ux-icon-close"
+            *ngIf="allowNull && selected"
+            (click)="resetValue($event)">
+        </i>
+
+        <i class="ux-select-icon ux-icon ux-icon-chevron-down ux-icon-down"></i>
+    </div>
+
+</button>
+
+<ux-menu #menu menuClass="select-menu">
+
+    <div [style.max-height.px]="maxHeight"
+         [style.width.px]="width">
+
+        <div [hidden]="busy">
+            <div [hidden]="hideFilter">
+
+                <input #filterInput
+                        type="text"
+                        [placeholder]="placeholder"
+                        class="filter-input"
```

Comment:
We might be able to use our `form-control` class on this element as well, which might avoid a lot of the custom styles you had to add and give you the correct focus styling etc.. and hopefully will allow it to pick up a lot of the micro focus styles for inputs when we add this to the MF site.

Reply:
Changed and committed.